SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/SpatialViewUtils.h

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "ComponentTestUtils.h"
+#include "SpatialView/ViewDelta.h"
+#include "SpatialView/WorkerView.h"
+
+namespace SpatialGDK
+{
+enum EntityStatus
+{
+	ADD,
+	REMOVE,
+	UPDATE
+};
+
+inline void AddComponentAddedChange(TArray<ComponentChange>& Changes, int ComponentId, double Value)
+{
+	const auto Data = Schema_CreateComponentData();
```

Comment:
There's a helper in `EntityComponentTestUtils` for creating data like this (and updates, for below)